keyboards/handwired/abnt2_83/keymaps/default/keymap.c

```diff
@@ -0,0 +1,273 @@
+        _______,    _______,  _______,          _______,                   _______,  _______,   _______,                  _______,  _______,  _______, _______,     _______
+    ),  
+ 
+[_PN] = LAYOUT(
+        _______,        _______,   _______,_______,_______,_______,_______,_______,_______,_______,   _______,  _______,  _______,  _______,  _______, _______,     _______,        _______, 
+        _______,        _______,   _______,_______,_______,_______,_______,_______,_______,_______,   _______,  _______,  _______,  _______,  _______, _______,     _______,        _______,   
+        _______,        _______,   _______,_______,_______,_______,_______,_______,_______,_______,   _______,  _______,  _______,            _______, _______,     _______,        _______, 
+        _______,        _______,   _______,_______,_______,_______,_______,_______,_______,_______,   _______,  _______,  _______,  _______,  _______, _______,     _______,        _______,
+        _______,    _______,  _______,          _______,                   _______,  _______,   _______,                  _______,  _______,  _______, _______,     _______
+    ),  
+};
+
+
+
+void led_set_user(uint8_t usb_led) {
+
+}
+void matrix_init_user(void) {
+  #ifdef USE_I2C
+    i2c_master_init();
+  #ifdef SSD1306OLED
+  // calls code for the SSD1306 OLED
+        _delay_ms(400);
+        TWI_Init(TWI_BIT_PRESCALE_1, TWI_BITLENGTH_FROM_FREQ(1, 800000));
+        iota_gfx_init();   // turns on the display
+  #endif
+  #endif
+    #ifdef AUDIO_ENABLE
+        startup_user();
+    #endif
+}
+
+void matrix_scan_user(void) {
+    #ifdef SSD1306OLED
+     iota_gfx_task();  // this is what updates the display continuously
+    #endif
+}
+
+void matrix_update(struct CharacterMatrix *dest, const struct CharacterMatrix *source)
+ {
+  if (memcmp(dest->display, source->display, sizeof(dest->display))) {
+    memcpy(dest->display, source->display, sizeof(dest->display));
+    dest->dirty = true;
+  }
+}
+
+void iota_gfx_task_user(void) {
+#if DEBUG_TO_SCREEN
+  if (debug_enable) {
+    return;
+  }
+#endif
+
+  struct CharacterMatrix matrix;
+
+  matrix_clear(&matrix);
+  matrix_write_P(&matrix, PSTR("ABNT2EX-83"));
+  
+  uint8_t layer = biton32(layer_state);
+
+  char buf[40];
+  snprintf(buf,sizeof(buf), "Undef-%d", layer);
+  matrix_write_P(&matrix, PSTR("\nLayer: "));
+  matrix_write(&matrix, layer_lookup[layer]);
+
+  // Host Keyboard LED Status
+  char led[40];
+    snprintf(led, sizeof(led), "\n\n%s  %s  %s",
+            (host_keyboard_leds() & (1<<USB_LED_NUM_LOCK)) ? "NUMLOCK" : "       ",
+            (host_keyboard_leds() & (1<<USB_LED_CAPS_LOCK)) ? "CAPS" : "    ",
+            (host_keyboard_leds() & (1<<USB_LED_SCROLL_LOCK)) ? "SCLK" : "    ");
+  matrix_write(&matrix, led);
+  matrix_update(&display, &matrix);
+}
+
+/* 
+void led_set_user(uint8_t usb_led) {
+
+  if (usb_led & (1 << USB_LED_NUM_LOCK)) {
+    
+  } else {
+    
+  }
+
```

Comment:
All of this should be removed, and replaced with the newer OLED Driver code: 
```suggestion
```

https://docs.qmk.fm/#/feature_oled_driver